streampark-flink/streampark-flink-packer/pom.xml

```diff
@@ -140,6 +140,13 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+
+        <!-- jackson-->
+        <dependency>
```

Comment:
It seems that this dependency is needed so that the dockerClient.inspectImageCmd(imageName).exec() code will not report an error.

Reply:
> It seems that this dependency is needed so that the dockerClient.inspectImageCmd(imageName).exec() code will not report an error.

Ok